.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+name: Build & Release
+on:
+  push:
+    - tag: 'v*'
+
+env:
+  GO_VERSION: "1.18.x"
+
+jobs:
+  build:
+    runs-on: "windows-2019"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - run: go build ./cmd/dmverity-vhd
+      - run: go build ./cmd/dmverity-vhd
+        env: 
+          GOOS: linux
+          GOARCH: amd64
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: binaries
+          path: |
+            dmverity-vhd.exe
+            dmverity-vhd
+
+  create_release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
+        with:
+          name: binaries
+          
+      - name: Publish release
+        uses: softprops/action-gh-release@v1
+        with:
+          prerelease: ${{ contains(github.ref, 'rc') }}
```

Comment:
does this mark a tag as pre-release? nit: `-rc`

Reply:
According to Bryce, that's how it works. And checking that it contains rc in the tag should be sufficient.

Reply:
This was taken directly from containerd.  Also, we leave without the "-", various forms continue to work, i.e. "vrc1.2.3", "v1.2.3-rc1", "v1.2.3rc1".

Reply:
sounds good.